src/test/java/org/aksw/simba/squirrel/fetcher/manage/SimpleOrderedFetcherManagerTest.java

```diff
@@ -0,0 +1,24 @@
+package org.aksw.simba.squirrel.fetcher.manage;
+
+import org.aksw.simba.squirrel.data.uri.CrawleableUri;
+import org.junit.Test;
+
+import java.io.File;
+import java.net.URI;
+
+import static org.junit.Assert.assertNull;
+
+public class SimpleOrderedFetcherManagerTest {
+
+    @Test
+    public void fetch() throws Exception{
+        CrawleableUri tempUri = new CrawleableUri(new URI("http://danbri.org/foaf.rdf"));
+        SimpleOrderedFetcherManager fetcher = new SimpleOrderedFetcherManager();
+        File data = fetcher.fetch(tempUri);
+        assertNull(data);
```

Comment:
This test will fail, right? Because your manager has no fetchers that it could use. You should have a very simple dummy fetcher class that does nothing but takes a flag in the constructor that tells it whether it should return a file if it is called. Additionally, it should have a flag that it sets as soon as it is asked to fetch something.

After that you have three test cases. In each test case, you create the manager and give two instances of the dummy fetcher class. 

1. both dummy fetchers should have a `true` flag. Results != null. 1st fetcher has been called. 2nd fetcher has not been called.
2. first fetcher has `false` while the second fetcher has a `true` flag. Result != null. Both fetchers have been called.
3. both dummy fetchers should have a `true` flag. Results == null. Both fetchers have been called.

Reply:
@MichaelRoeder - Created a dummy fetcher class and written three testcases to test the possible conditions as mentioned above by passing the two instances of dummy fetcher class to SimpleOrderedFetcherManager.